lib/jsdom/browser/Window.js

```diff
@@ -193,6 +202,10 @@ function Window(options) {
           return this.responseText;
         } else if (this.responseType === "json") {
           return JSON.parse(this.responseText);
+        } else if (window.document.location.protocol === "file:") {
+          // node-xhr doesn't properly set the responseType for file requests.
+          // return the text anyway.
```

Comment:
But if the type is set then it falls into one of the two conditionals above. In this case the `responseType` isn't being set and xhr isn't properly defaulting it for file urls.